specification/redis/resource-manager/readme.md

```diff
@@ -26,18 +26,19 @@ These are the global settings for the Redis API.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2019-07
+tag: package-2019-07-preview
 ```
 
 
-### Tag: package-preview-2019-07
+### Tag: package-2019-07-preview
 
-These settings apply only when `--tag=package-preview-2019-07` is specified on the command line.
+These settings apply only when `--tag=package-2019-07-preview` is specified on the command line.
 
-```yaml $(tag) == 'package-preview-2019-07'
+```yaml $(tag) == 'package-2019-07-preview'
```

Comment:
Just curious, is the api version of  2019-07 still in preview?

Reply:
yes, the -preview postfix was already removed in the service, but the service team still wants to keep it under "preview" folder, not under stable